src/Behat/Cores/Drupal8.php

```diff
@@ -469,4 +469,23 @@ public function createFileUrl($file, bool $relative = TRUE) {
     throw new InvalidArgumentException('%s method only accept %s objects in Drupal 8 or higher', __METHOD__, FileInterface::class);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getLanguagePrefix($language) {
+    $language_manager = \Drupal::languageManager();
+    $language_list = $language_manager->getStandardLanguageList();
+    array_walk($language_list, function (&$item1, $key) {
```

Comment:
In this way, if there are two words to define a language, only the first would be used.

Such an example, if Spanish can be selected as 'Spanish' or 'Español' (see [here](https://git.drupalcode.org/project/drupal/-/blob/9.4.x/core/lib/Drupal/Core/Language/LanguageManager.php#L254)), only Spanish can be used in the step. If there are multiple ways to write the language label we must allow all so testers are free to write it in an intuitive manner.

I suggest building a key value array ($language_list_map) with the key being the label and the value being the lancode, something like this:

```php
  [
     'Spanish' => 'es',
     'Español => 'es',
    ...
  ]
```
Then you can use that variable in the line 482:

```php
     $langcode = $language_list_map[$language] ?? '';
```